drivers/rpmsg/Kconfig

```diff
@@ -15,6 +15,18 @@ config RPMSG_CHAR
 	  in /dev. They make it possible for user-space programs to send and
 	  receive rpmsg packets.
 
+config RPMSG_SYSLOG
+	tristate "RPMSG syslog redirection"
+	depends on RPMSG
+	help
+	  Say Y here to redirect the syslog/printf from remote processor into
+	  the kernel log which is very useful to see what happen in the remote
+	  side.
+
+	  If the remote processor hang during bootup or panic at the runtime,
+	  We can even cat /sys/kernel/debug/remoteproc/remoteprocX/trace0 to
+	  get the last log which doesn't output yet.
+
```

Comment:
this isn't very clear and tells me that the actual usage is simpler and more complete if you directly access to the trace buffer of the remote processor.

Reply:
Since the buffer in DSP/MCU is limited, rpmsg-syslog client will send the log to kernel regularly to avoid loss the log.
But if DSP/MCU suddenly panic or watchdog reset, some log may still in the buffer, we can get the critical information from /sys/kernel/debug/remoteproc/remoteprocX/trace0 in this case.
![syslog](https://user-images.githubusercontent.com/18066964/47409793-6edf6300-d796-11e8-8e6d-325db613d082.JPG)